src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
-'use strict';
+"use strict";
 
-// write your code here
+const div = document.createElement("div");
```

Comment:
The created div element is shared by both success and error handlers. It may create unwanted side effects when both promises get resolved/rejected simultaneously.